packages/react-aria-components/scripts/create-react-aria-components/templates/vanilla-css/components/Button/Button.mdx

```diff
@@ -0,0 +1,11 @@
+import {Button} from './Button'
```

Comment:
why include the mdx? storybook docs eventually?

Reply:
Yes, exactly